Fixed GroupForm bug

JosteinLindhom opened the pull request:
Fixes #982.

meling commented:
This is okay, but maybe we should add a toggle button:
- students only
- teachers only

JosteinLindhom commented:
> This is okay, but maybe we should add a toggle button:
> 
> * students only
> * teachers only

Added a toggle for teachers. I also fixed a bug where students would not show in the list of available users if removed from a group.

meling reviewed (approved):
LGTM
